docs/marketplace/user/intro-to-spryker/back-office-for-marketplace-operator.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Back Office for Marketplace Operator
+description: This document describes how a Marketplace Operator works in the Back Office.
+template: concept-topic-template
+---
+
+A *Marketplace Operator* is a company that owns the marketplace. In a marketplace business model, one or more of such a company's employees represent the marketplace operator and manage the store. We call such users as Marketplace Administrator <!---LINK-->. Depending on the marketplace model, the marketplace operator either manages only other merchants' orders or also sells products as a main merchant <!---LINK-->).  
+
+## Merchant management
+
+Merchant accounts are created in the Back Office, and a Marketplace Administrator <!---LINK--> is required to facilitate their creation. In the **Marketplace** > **Merchants** section of the Back Office, a marketplace operator manages merchants, that is:
+
+* Creates merchants
+* Edits merchants
+* Approves and denies merchants
+* Activates and deactivates merchants
+
+For detailed instructions on managing merchants, see [Managing merchants](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchants.html).
+
+## Offer management
+
+In a marketplace, multiple merchants can sell the same product. That's why, instead of creating duplicate products, merchants create offers. Only the offers approved by a marketplace administrator are displayed in the Storefront. In the **Marketplace** > **Offers** section of the Back Office, a marketplace operator manages offers as follows:
```

Comment:
once we say "marketplace administrator", just below "operator". should we simplify it somehow?

Reply:
updated to marketplace administrator